nighthawk/source/client/output.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+import "nighthawk/source/client/options.proto";
+
+
+message Percentile {
+    google.protobuf.Duration latency = 1;
+    double percentile = 2;
+    int64 count = 3;
+    double inverted_percentile = 4;
```

Comment:
Can you add comments to these? Mostly self-explanatory, but inverted percentile isn't..

Reply:
I removed inverted_percentile, which actually just was `1/(1-Percentile)` and therefore redundant.